integration/liboqs/ML-DSA-44_META.yml

```diff
@@ -10,56 +10,71 @@ length-signature: 2420
-  - name: x86_64
-    version: FIPS204
-    folder_name: .
-    compile_opts: -DMLDSA_MODE=2 -DMLD_CONFIG_FILE="../integration/liboqs/config_x86_64.h"
-    signature_keypair: PQCP_MLDSA_NATIVE_MLDSA44_X86_64_keypair
-    signature_signature: PQCP_MLDSA_NATIVE_MLDSA44_X86_64_signature
-    signature_verify: PQCP_MLDSA_NATIVE_MLDSA44_X86_64_verify
-    api-with-context-string: true
-    sources: integration/liboqs/config_x86_64.h integration/liboqs/fips202_glue.h integration/liboqs/fips202x4_glue.h mldsa/cbmc.h mldsa/common.h mldsa/ct.c mldsa/ct.h mldsa/debug.c mldsa/debug.h mldsa/native/api.h mldsa/native/meta.h mldsa/native/x86_64 mldsa/ntt.c mldsa/ntt.h mldsa/packing.c mldsa/packing.h mldsa/params.h mldsa/poly.c mldsa/poly.h mldsa/polyvec.c mldsa/polyvec.h mldsa/randombytes.h mldsa/reduce.h mldsa/rounding.h mldsa/sign.c mldsa/sign.h mldsa/symmetric.h mldsa/sys.h mldsa/zetas.inc
-    supported_platforms:
-      - architecture: x86_64
-        operating_systems:
-          - Darwin
-          - Linux
-        required_flags:
-          - avx2
-          - bmi2
-          - popcnt
-  - name: aarch64
-    version: FIPS204
-    folder_name: .
-    compile_opts: -DMLDSA_MODE=2 -DMLD_CONFIG_FILE="../integration/liboqs/config_aarch64.h"
-    signature_keypair: PQCP_MLDSA_NATIVE_MLDSA44_AARCH64_keypair
-    signature_signature: PQCP_MLDSA_NATIVE_MLDSA44_AARCH64_signature
-    signature_verify: PQCP_MLDSA_NATIVE_MLDSA44_AARCH64_verify
-    api-with-context-string: true
-    sources: integration/liboqs/config_aarch64.h integration/liboqs/fips202_glue.h integration/liboqs/fips202x4_glue.h mldsa/cbmc.h mldsa/common.h mldsa/ct.c mldsa/ct.h mldsa/debug.c mldsa/debug.h mldsa/native/api.h mldsa/native/meta.h mldsa/native/aarch64 mldsa/ntt.c mldsa/ntt.h mldsa/packing.c mldsa/packing.h mldsa/params.h mldsa/poly.c mldsa/poly.h mldsa/polyvec.c mldsa/polyvec.h mldsa/randombytes.h mldsa/reduce.h mldsa/rounding.h mldsa/sign.c mldsa/sign.h mldsa/symmetric.h mldsa/sys.h mldsa/zetas.inc
-    supported_platforms:
-      - architecture: arm_8
-        operating_systems:
-            - Linux
-            - Darwin
-        required_flags:
-            - asimd
+- name: ref
+  version: FIPS204
+  folder_name: .
+  compile_opts: -DMLDSA_MODE=2 -DMLD_CONFIG_FILE="../integration/liboqs/config_c.h"
+  signature_keypair: PQCP_MLDSA_NATIVE_MLDSA44_C_keypair
+  signature_signature: PQCP_MLDSA_NATIVE_MLDSA44_C_signature
+  signature_verify: PQCP_MLDSA_NATIVE_MLDSA44_C_verify
+  api-with-context-string: true
+  sources: integration/liboqs/config_c.h integration/liboqs/fips202_glue.h integration/liboqs/fips202x4_glue.h
+    mldsa/cbmc.h mldsa/common.h mldsa/ct.c mldsa/ct.h mldsa/debug.c mldsa/debug.h
+    mldsa/native/api.h mldsa/native/meta.h mldsa/ntt.c mldsa/ntt.h mldsa/packing.c
```

Comment:
`mldsa/native/api.h` and `mldsa/native/meta.h ` should not be included here as these are only needed for the native implementations. Please remove.